test/Microsoft.AspNetCore.Razor.Test.Common/Language/Legacy/SyntaxTreeNodeVerifier.cs

```diff
@@ -0,0 +1,276 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Xunit;
+using Xunit.Sdk;
+
+namespace Microsoft.AspNetCore.Razor.Language.Legacy
+{
+    public static class SyntaxTreeNodeVerifier
```

Comment:
Most of the code in this file is from `IntermediateNodeVerifier`. I'll later think about sharing some of this code.

Reply:
Share what makes sense. Code reuse isn't an end goal in itself 😆